client/my-sites/plugins/access-control.js

```diff
@@ -31,6 +31,10 @@ const getWpcomPluginPageError = () => {
 const hasRestrictedAccess = ( site ) => {
 	site = site || sites.getSelectedSite();
 
+	if ( site.options && site.options.is_automated_transfer ) {
+		return;
```

Comment:
No need to refactor, since it's already doing strange things, but I just want to point out that I would normally expect a function with this name to always return a boolean value. 

Let's keep in mind that we should try to craft functions to always return a value, or nothing (say like a full side effects function), where we'd avoid the latter if possible.